### ISSUE TITLE
APIView needs to support different target framework versions (TFMs) for .NET

### ISSUE THREAD

**Issue by heaths** [Azure/azure-sdk-tools#5137]

**Comment by praveenkuttappan** [comment]
This will be change in parser side if the solution is to show a front matter to indicate applicable TFM.

**Comment by heaths** [comment]
Effectively blocked on #5374

**Comment by KrzysztofCwalina** [comment]
> we'll need APIView to support different public APIs

I don't think we are planning to have different APIs based on the TFM.

**Comment by JoshLove-msft** [comment]
> > we'll need APIView to support different public APIs
> 
> I don't think we are planning to have different APIs based on the TFM.

That is my understanding as well. IIRC @tg-msft was looking for tooling to detect API changes between TFMs so we can prevent shipping.

**Comment by heaths** [comment]
Yes, and I already dup'd his issue to Azure/azure-sdk-for-net#33433. At some point we may have to - and already have for Azure.Core because of `Activity` support - have differences. If we're not interested in seeing those differences we can close this, but the bug I linked to the tools repo serves other purposes.

**Comment by care03324-spec** [comment]
https://www.66lottery.vip/#/pages/games/wingo/wingo
